Makefile

```diff
@@ -0,0 +1,19 @@
+
+EPOCH_TEST_COMMIT	:= 91d6d8466e68f1efff7977b63ad6f48e72245e05
+
+test: .gitvalidation
+
+# When this is running in travis, it will only check the travis commit range
+.gitvalidation:
+	@which git-validation > /dev/null 2>/dev/null || (echo "ERROR: git-validation not found. Consider 'make install.tools' target" && false)
```

Comment:
`which` [is not in POSIX][1], but [`command` is][2].  So you can achieve this same functionality more portably with:

    @command -v git-validation >/dev/null 2>/dev/null || (echo "ERROR: git-validation not found. Consider 'make install.tools' target" && false)

You can also use `2>&` instead of `2>/dev/null` (docs [here][3]).  This part isn't a portability issue though, so I'm fine either way.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/idx/utilities.html
[2]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html
[3]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_06